### PR TITLE
Doc: GSG update for ACRN Industry Scenario with Ubuntu Service VM

### PR DESCRIPTION
**Pull request by fuzhongl**
Update Getting Started Guide for ACRN Industry Scenario with Ubuntu Service VM

Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/getting-started/building-from-source.rst [hunk 1]
```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
...
-As mentioned in :ref:`ACRN Configuration Tool <vm_config_workflow>`, the
-Board configuration and VM configuration can be imported from XML files.
-If you want to build the hypervisor with XML configuration files,
-specify the file location as follows (assuming you're at the top level
-of the acrn-hypervisor directory):
-
-.. code-block:: none
-
-   $ make BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
+   $ make all OARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
```
Comment (gvancuts) [inline doc/getting-started/building-from-source.rst:260]:
```suggestion
   $ make all BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
```

File: doc/getting-started/building-from-source.rst [hunk 2]
```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
 
 The build results are found in the ``build`` directory. You can specify
 a different Output folder by setting the ``O`` ``make`` parameter,
```
Comment (gvancuts) [inline doc/getting-started/building-from-source.rst:254]:
```suggestion
a different build folder by setting the ``O`` ``make`` parameter,
```
We should call this a `build` folder instead IMHO, @dbkinder , thoughts?

File: doc/getting-started/building-from-source.rst [hunk 3]
```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
...
-of the acrn-hypervisor directory):
-
-.. code-block:: none
-
-   $ make BOARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
+   $ make all OARD_FILE=$PWD/misc/vm_configs/xmls/board-xmls/nuc7i7dnb.xml \
    SCENARIO_FILE=$PWD/misc/vm_configs/xmls/config-xmls/nuc7i7dnb/industry.xml TARGET_DIR=xxx
 
+The build results are found in the ``build`` directory. You can specify
+a different Output folder by setting the ``O`` ``make`` parameter,
```
Comment (gvancuts) [inline doc/getting-started/building-from-source.rst:264]:
```suggestion
a different build folder by setting the ``O`` ``make`` parameter,
```

File: doc/getting-started/building-from-source.rst [hunk 4]
```diff
@@ -252,29 +252,18 @@ Now you can build all these components at once as follows:
...
+for example: ``make O=build-nuc``.
```
Comment (gvancuts) [inline doc/getting-started/building-from-source.rst:265]: Can you also briefly explain what the `TARGET_DIR` parameter does?

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 5]
```diff
@@ -477,7 +451,8 @@ Launch the RTVM
 
   .. code-block:: none
 
-     $ sudo /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+     $ cp /home/acrn/work/acrn-hyperviso/misc/vm_configs/sample_launch_scripts/nuc/launch_hard_rt_vm.sh  /usr/share/acrn/
```
Comment (gvancuts) [inline doc/getting-started/rt_industry_ubuntu.rst:454]: I believe you'll need to be `sudo` here?

**Review by gvancuts — CHANGES_REQUESTED** [review]

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/getting-started/building-from-source.rst (hunk 4 above)
Comment (fuzhongl) [reply doc/getting-started/building-from-source.rst:265]: @gvancuts  The  TARGET_DIR parameter is explained in Note:
The TARGET_DIR parameter specifies what directory is used to store configuration files imported from XML files. If the TARGET_DIR is not specified, the original configuration files of acrn-hypervisor would be overridden.

File: doc/getting-started/building-from-source.rst (hunk 3 above)
Comment (fuzhongl) [reply doc/getting-started/building-from-source.rst]: Done!

File: doc/getting-started/building-from-source.rst (hunk 2 above)
Comment (fuzhongl) [reply doc/getting-started/building-from-source.rst:254]: Done!

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 5 above)
Comment (fuzhongl) [reply doc/getting-started/rt_industry_ubuntu.rst]: Thanks! Done!

File: doc/getting-started/building-from-source.rst (hunk 1 above)
Comment (fuzhongl) [reply doc/getting-started/building-from-source.rst]: Thanks! Done!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/getting-started/rt_industry_ubuntu.rst [hunk 6]
```diff
@@ -223,7 +223,7 @@ Build and install the ACRN kernel
 
    .. code-block:: none
 
-      $ git checkout -b v2.2 remotes/origin/release_2.2
+      $ git checkout v2.3
```
Comment (gvancuts) [inline doc/getting-started/rt_industry_ubuntu.rst:226]: Will this tag be created later? Or should the user check the `release_2.3` branch?

**Review by gvancuts — COMMENTED** [review]
Thanks! There is some more clean-up that I'd like to see:
* Remove the dependency on `gnu-efi` (when we install the build tools and deps)
* Remove this line as part of the `iasl` instructions (`sudo -E apt-get install iasl`)
* Remove the description of the `misc/efi-stub` folder which does not exist anymore
* I believe the gcc version in Ubuntu 18.04 is now **7.5**

File: doc/getting-started/rt_industry_ubuntu.rst (hunk 6 above)
Comment (dbkinder) [reply doc/getting-started/rt_industry_ubuntu.rst:226]: v2.3 will be the "official release" tag, so we should be using that.

**Comment by fuzhongl** [comment]
@gvancuts 
 misc/efi-stub folder was already removed in code.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by gvancuts — COMMENTED** [review]
The "Building from source" doc still has a couple of references to the EFI stuff:
* `gnu-efi` as a build dependency
* Reference to the `efi-stub` folder

**Comment by dbkinder** [comment]
@NanlinXie @gvancuts comments still need to be resolved for the v2.3 release

**Comment by fuzhongl** [comment]
> The "Building from source" doc still has a couple of references to the EFI stuff:
> 
> * `gnu-efi` as a build dependency
> * Reference to the `efi-stub` folder

@gvancuts 
Remove them in "Building from source".
Thanks!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Review by gvancuts — APPROVED** [review]
Thanks @fuzhongl !

**Review by dbkinder — APPROVED** [review]

**Pull request merged**